src/pages/diagram/dialogs/confirmation-delete/styles.ts

```diff
@@ -0,0 +1,54 @@
+import styled from "styled-components";
+
+import Button from "@mui/material/Button";
+import Dialog from "@mui/material/Dialog";
+import DialogTitle from "@mui/material/DialogTitle";
+
+import CloseIcon from "@mui/icons-material/Close";
+
+import { IconButton } from "shared/components/icon-button";
+import { Divider } from "@mui/material";
+
+export const ConfirmationDeleteWrapper = styled(Dialog).attrs({

```

Comment:
Acredito que ficaria mais fácil vc adicionar esses atributos direto no componente, assim não precisa declarar o componente no styles. Só usar o componente com styled components quando for estilizar o componente.

Reply:
Entendi. Estamos fazendo dessa forma para manter o padrão do projeto e deixar toda a importação de estilo na página de styles.